src/commctl/cli.py

```diff
@@ -633,6 +636,9 @@ def add_cluster_commands(argument_parser):
     # Sub-command: cluster create
     verb_parser = subject_subparser.add_parser('create')
     verb_parser.required = True
+    verb_parser.add_argument(
+        'type', help='Type of the cluster',
+        choices=('kubernetes', 'host_only'))
```

Comment:
After sleeping on it I think your feeling is right. The API can still require a ``type`` but ``commctl`` can use an option and use a sane default. I think that really ends up being a better UX for the user. Having it as a positional looks like it could he confusing.